src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -156,7 +156,8 @@ private void Update()
 			UpdateTextTrimming();
 			UpdateTextFormatted();
 			UpdatePaint();
-		}
+            UpdateMaxLines();
```

Comment:
indentation

Reply:
I know that it looks like the indentation is wrong here, but that is just the way github presents the information, if you go see the file, the indentation is fine.
https://github.com/unoplatform/uno/blob/85d547691ae52af9c50d98b00e9dd227389061f2/src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs#L159

Reply:
in fact it seems to be using spaces. Adjust it to use tabs.